bundle-workflow/Jenkinsfile

```diff
@@ -57,14 +49,15 @@ pipeline {
                     post() {
                         always {
                             cleanWs disableDeferredWipeout: true, deleteDirs: true
-                            sh "rm -rf ~/.m2/"
                         }
                     }
                 }
                 stage('build-x86') {
                     agent {
-                        node {
-                            label 'Jenkins-Agent-al2-x64-m5xlarge'
+                        docker {
+                            label 'Jenkins-Agent-al2-x64-c54xlarge-Docker-Host'
```

Comment:
This is a sizeable host difference, does this build better/faster?

Reply:
This is a c54xlarge instance that specifically setup for docker, it can provision 10 docker containers per host and have enough power to run the builds.